pkg/scheduler/framework/plugins/nodeports/node_ports.go

```diff
@@ -76,6 +76,10 @@ func getContainerPorts(pods ...*v1.Pod) []*v1.ContainerPort {
 // PreFilter invoked at the prefilter extension point.
 func (pl *NodePorts) PreFilter(ctx context.Context, cycleState *framework.CycleState, pod *v1.Pod) (*framework.PreFilterResult, *framework.Status) {
 	s := getContainerPorts(pod)
```

Comment:
@tangwz 
Can you take a look at this suggestion from @/ahg-g?
https://github.com/kubernetes/kubernetes/pull/115318/files#r1169034360

Reply:
Yes, lets optimize this further, we only care about host ports.

Reply:
> 

I think `CheckConflict(ip, protocol string, port int32)` need we return HostIP, Protocol and HostPort as the parameter, I don't understand how we only return host ports?